mappings/net/minecraft/world/PersistentStateManager.mapping

```diff
@@ -8,10 +8,20 @@ CLASS net/minecraft/class_26 net/minecraft/world/PersistentStateManager
 		ARG 2 dataFixer
 	METHOD method_120 get (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
```

Comment:
This method is specifically about the loading process, and it logs `"Error loading saved data"`. I would call it `load` or `loadSavedState`.

Reply:
https://github.com/FabricMC/yarn/pull/891/commits/e84a8b130c3161022e8941e12ce77883b50cc3bc